legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -138,7 +138,7 @@ def post_businesses():
         return {'error': babel('Unable to create {0} Filing.'.format(Filing.FILINGS[filing_type]['title']))}, \
             HTTPStatus.SERVICE_UNAVAILABLE
 
-    return ListFilingResource.put(bootstrap.identifier, None)
+    return saving_filings(identifier=bootstrap.identifier)  # pylint: disable=no-value-for-parameter
```

Comment:
I think this might be problematic if we do it this way?  

We bypass some checks that was originally done in the `PUT` call to the `ListingFilingResource`.  

Also, it seems like it was possible via this endpoint to do a `POST` and create the business in one go if the "draft" query param wasn't set?  I think that's what I'm seeing.  With this update, we lose the ability to do that.

Reply:
ListFilingResource.put in v1 and saving_filings in v2 are same. It won't bypass any checks

Reply:
oh sorry, I got confused.  I was looking at the `ListFilingResource.save_filing`